code/modules/projectiles/projectile/beams.dm

```diff
@@ -166,6 +166,15 @@
 	tracer_type = /obj/effect/projectile/tracer/stun
 	impact_type = /obj/effect/projectile/impact/stun
 
+/obj/item/projectile/beam/stun/on_impact(var/atom/A)

```

Comment:
35% is absolutely massive for how many shots an energy gun/taser has and how fast they can deploy those shots. Not a necessary change.